src/NumoBundle/Resources/views/site/profilMember.html.twig

```diff
@@ -4,155 +4,125 @@
+            <div class="separation"></div>
+            <h1>{{ user.name }} {{ user.firstname }}</h1>
+            <div class="separation"></div>
         </div>
-    </div>
-
-    <div class="row iconContact">
-        {% if user.facebook != null %}
-            <div class="col-lg-offset-1 col-lg-1">
-                <a target="_blank" href="{{ user.facebook }}">
-                <img src="{{ asset('/img/rs-facebook.png') }}" alt="facebook" class="rs-button"/>
-                </a>
-            </div>
-            <div class="col-lg-1">
-                <h3>{{ user.facebook }}</h3>
-            </div>
-        {% endif %}
 
-        {% if user.twitter != null %}
-            <div class="col-lg-offset-1 col-lg-1">
-                <a href="{{ user.twitter }}" target="_blank">
-                    <img src="{{ asset('/img/rs-twitter.png') }}" alt="twitter" class="rs-button"/>
-                </a>
-            </div>
-            <div class="col-lg-1">
-                <h3><a href="{{ user.twitter }}" target="_blank"></a>{{ user.twitter }}</h3>
+        <div class="row">
+            <div class="col-lg-offset-1 col-lg-2">
+                {% if user.imageUrl == null %}
+                    <img src="{{ asset('img/person_placeholder.png') }}" alt="placeholder" style="height: 231px">
+                {% else %}
+                    <img src="{{ asset('uploads/users/' ~ user.imageUrl.filename) }}" alt="{{ user.name }}"
+                         style="height: 70px">
+                {% endif %}
             </div>
-        {% endif %}
 
-        {% if user.linkedin != null %}
-            <div class="col-lg-offset-1 col-lg-1">
-                <a href="{{ user.linkedin }}" target="_blank">
-                    <img src="{{ asset('/img/rs-linkedin.png') }}" alt="linkedin" class="rs-button"/>
-                </a>
-            </div>
-            <div class="col-lg-1">
-                <h3>{{ user.linkedin }}</h3>
-            </div>
-        {% endif %}
-    </div>
-    <div class="row iconContact">
-        {% if user.email != null %}
-            <div class="col-lg-offset-1 col-lg-1">
-                <img src="{{ asset('/img/mail.png') }}" alt="email" class="rs-button"/>
+            <div class="col-lg-5">
+                <div class="row">
+                    <h2>{{ user.website }}</h2>
+                    <p>{{ user.description }}</p>
+                </div>
             </div>
-            <div class="col-lg-1">
-                <h3>{{ user.email }}</h3>
-            </div>
-        {% endif %}
+        </div>
 
-        {% if user.phone != null %}
-            <div class="col-lg-offset-1 col-lg-1">
-                <img src="{{ asset('/img/tel.png') }}" alt="telephone" class="rs-button"/>
-            </div>
-            <div class="col-lg-2">
-                <h3>{{ user.phone }}</h3>
-            </div>
-        {% endif %}
+        <div class="row iconContact">
+            {% if user.facebook != null %}
```

Comment:
vu qu'on a 3 fois le meme code pour les 3 réseaux sociaux, on pourrait sans doute envisager une boucle dans twig : 
for (reseauSocial in ['facebook', 'twitter', 'linkedin']) {
...
}